diff -Nuar x/boost/program_options/errors.hpp y/boost/program_options/errors.hpp
--- x/boost/program_options/errors.hpp	2014-09-26 17:30:47.000000000 -0500
+++ y/boost/program_options/errors.hpp	2014-09-26 17:33:23.099691445 -0500
@@ -26,7 +26,7 @@
     inline std::string strip_prefixes(const std::string& text)
     {
         // "--foo-bar" -> "foo-bar"
-        return text.substr(text.find_first_not_of("-/"));
+        return text.substr(text.find_last_of("-/"));
     }
 
     /** Base class for all errors in the library. */
